### DISCUSSION TITLE
Split payments

### DISCUSSION THREAD

**Discussion by jigarzon** [btcpayserver/btcpayserver#3151] [Technical Support] [answered]
I'm a developer and I was looking for a way to implement split payments. The use case is: for every payment received by the store XX, I want the store YY to receive a % of the payment. Can I do that with BTCpayserver?

Thanks in advance!

**Comment by britttttk** [comment, marked as answer]
It's possible with transmuter's onchain forwarder preset https://docs.btcpayserver.org/Transmuter/#what-preset-recipes-are-provided
